loopchain/channel/channel_statemachine.py

```diff
@@ -116,7 +116,9 @@ def switch_role(self):
         pass
 
     def _is_leader(self):
-        return self.__channel_service.block_manager.peer_type == loopchain_pb2.BLOCK_GENERATOR
+        return \
+            ChannelProperty().node_type == conf.NodeType.CommunityNode and \
+            self.__channel_service.block_manager.peer_type == loopchain_pb2.BLOCK_GENERATOR
```

Comment:
This would be better for readability

```py
def _is_leader(self):
    return (ChannelProperty().node_type == conf.NodeType.CommunityNode 
            and self.__channel_service.block_manager.peer_type == loopchain_pb2.BLOCK_GENERATOR)
```